src/components/molecules/DcGameCard.vue

```diff
@@ -0,0 +1,49 @@
+<script setup lang="ts">
+import { computed } from "vue";
+import { Plataform } from "~/types";
+
+const props = defineProps({
+  img: String,
+  name: String,
+  platforms: Array<Plataform>,
+  price: Number,
+  promotion: Number,
+});
+
+const getPromotionText = computed(
+  () => props?.promotion && `${props.promotion * 100}%`
+);
+
+const hasWindows = computed(() =>
+  props.platforms?.find((plataform) => plataform.name.includes("Win"))
+);
+const hasSteam = computed(() =>
+  props.platforms?.find((plataform) => plataform.name.includes("Steam"))
+);
+</script>
+
+<template>
+  <QCard>
+    <QImg :src="img" :aspect-ratio="3 / 1" />
+
+    <QCardSection>
+      <div>
+        <h3 class="text-body2">Lawn Mowing Simulator</h3>
+      </div>
+      <div class="row q-gutter-x-xs">
+        <QIcon
+          v-if="hasWindows"
+          name="i-mdi-microsoft-windows"
+          title="Windows"
+        ></QIcon>
+        <QIcon v-if="hasSteam" name="i-mdi-steam" title="Steam"></QIcon>
```

Comment:
Here I noticed a pattern were the only variation is the title and the suffix of the platform.
Now instead of have one v-if and one computed for every single gaming platform, we could optmize this creating one single computed that maps every key string the platforms Array property, to have a new Array of objects with the desired props. Then we would have a v-for to iterate over the list dynamically.

Reply:
Moved this into a computed property called platforms